data/coalition jobs.txt

```diff
@@ -63,7 +63,7 @@ mission "Coalition: Ringworld Workers 3"
 		"coalition jobs" ++
 		payment
 		payment 20000
-		dialog `You drop off the volunteers on the <planet>. A few minutes later you receive a transfer of <payment>.`
+		dialog `You drop off the volunteers on the <planet>. A few minutes later, you receive a transfer of <payment>.`
```

Comment:
Ah, I forgot that commas can be nothing more than a style choice sometimes.If MZ's writing style would not have these commas, then I'll remove them later today.